### PR TITLE
Update release version v0.37.0 (#7817)

### PR DESCRIPTION
**Pull request by vishwahiremat**
# Description

Update release version v0.37.0

## Type of change

- This pull request fixes a bug in Radius and has an approved issue (issue link required).
- This pull request adds or changes features of Radius and has an approved issue (issue link required).
- This pull request is a minor refactor, code cleanup, test improvement, or other maintenance task and doesn't change the functionality of Radius (issue link optional).

Fixes: #issue_number

---------

Signed-off-by: Vishwanath Hiremath <vhiremath@microsoft.com>
(cherry picked from commit 65ca82d4004e78ad412d7415f998f1b3bd0fc5b7)

# Description

_Please explain the changes you've made._

## Type of change

- This pull request fixes a bug in Radius and has an approved issue (issue link required).
- This pull request adds or changes features of Radius and has an approved issue (issue link required).
- This pull request is a minor refactor, code cleanup, test improvement, or other maintenance task and doesn't change the functionality of Radius (issue link optional).

Fixes: #issue_number

### CODE REVIEW

**Review by ytimocin — APPROVED** [review]

File: docs/release-notes/v0.37.0.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+## Breaking changes
+
+- **Radius merge to official Bicep** introduces some breaking changes. To use Radius features with the official Bicep compiler please do the following:
+    1. Create a bicepconfig.json with the extensions to author and deploy Radius and AWS types. More information [here](https://docs.radapp.io/guides/tooling/bicepconfig/overview/).
+    2. Update the import statements in the existing bicep files
+        - `import radius as radius` should become `extension radius` to use Radius types
+        - `import aws as aws` should become `extension aws` to use AWS types
+        - `import kubernetes as kubernetes {}` should become `extension kubernetes with {} as kubernetes to use Kubernetes types`
+    3. If you have Radius-Bicep extension installed, please disable and install the official bicep extension. More details [here](https://edge.docs.radapp.io/guides/tooling/vscode/howto-vscode-bicep/).
```
Comment (Reshrahim) [inline docs/release-notes/v0.37.0.md:30]:
```suggestion
    3. If you have Radius-Bicep extension installed, please disable and install the official bicep extension. More details [here](https://docs.radapp.io/guides/tooling/vscode/howto-vscode-bicep/).
```

File: docs/release-notes/v0.37.0.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+### Radius AWS Cloud provider supports IAM Roles for Service Accounts (IRSA)
+You can now configure your AWS cloud provider to use AWS federated identity (IRSA) to deploy and manage AWS resources. With this support, infrastructure operators are not burdened with the rotation of the credentials anymore. More information on [how to configure the AWS IRSA](https://docs.radapp.io/guides/operations/providers/aws-provider/howto-aws-provider-irsa/)
+
+### Test Improvements
+We separated the cloud tests and non-cloud tests in our functional  test workflows to improve the test verification process for contributors. In the new workflow the non-cloud test workflows will trigger automatically when pull requests are submitted, while tests using cloud resources will need approval from approvers/maintainers to be triggered. We also fixed an issue with `magpiego` image failing for MacOS+M1 chip users by publishing a multi-architectural image.
```
Comment (Reshrahim) [inline docs/release-notes/v0.37.0.md:20]:
```suggestion
We separated the cloud tests and non-cloud tests in our functional  test workflows to improve the test verification process for contributors. In the new workflow the non-cloud test workflows will trigger automatically when pull requests are submitted, while tests using cloud resources will need approval from approvers/maintainers to be triggered. We also fixed an issue with `magpiego` image failling for MacOS+M1 chip users by publishing a multi-architectural image.
```

File: docs/release-notes/v0.37.0.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+We would like to extend our thanks to all the [new](#new-contributors) and existing contributors who helped make this release possible!
+
+## Intro to Radius
+
+If you're new to Radius, check out our website, [radapp.io](https://radapp.io), for more information. Also visit our [getting started guide](https://docs.radapp.io/getting-started/) to learn how to install Radius and create your first app.
+
+## Highlights
+
+### Radius is merged with the official Bicep
+Previously Radius used a fork of the Bicep to author and deploy Radius and AWS types. With 0.37 release we have merged our extensibility updates with the upstream and are deprecating the temporary fork of Bicep. To use Radius features with the official Bicep compiler, you need to create a [`bicepconfig.json`]((https://docs.radapp.io/guides/tooling/bicepconfig/overview/)) with the extensions to author and deploy Radius and AWS types. Please checkout the breaking changes section for detailed guidance.
```
Comment (Reshrahim) [inline docs/release-notes/v0.37.0.md:14]:
```suggestion
Previously Radius used a fork of the Bicep to author and deploy Radius and AWS types. With 0.37 release we have merged our extensibility updates with the upstream and are deprecating the temporary fork of Bicep. To use Radius features with the official Bicep compiler, you need to create a [`bicepconfig.json`](https://docs.radapp.io/guides/tooling/bicepconfig/overview/) with the extensions to author and deploy Radius and AWS types. Please checkout the breaking changes section for detailed guidance.
```

File: docs/release-notes/v0.37.0.md (hunk 2 above)
Comment (vishwahiremat) [reply docs/release-notes/v0.37.0.md:20]: is it a typo? do we need a fix here?

Comment (Reshrahim) [reply docs/release-notes/v0.37.0.md:20]: "failling" is incorrectly spelled

**Review by Reshrahim — APPROVED** [review]

**Pull request merged**